jpa/WFLY-19554_cdi_persistence_ee11.adoc

```diff
@@ -0,0 +1,250 @@
+---
+categories:
+  - core
+  - persistence
+stability-level:
+  - community
+issue:
+https://issues.redhat.com/browse/WFLY-19554
+
+
+feature-team:
+ developer: https://github.com/scottmarlow/
+ sme: https://github.com/starksm64
+ sme: https://github.com/gavinking 
+promoted-by:
+:author:            Scott Marlow
+:email:             smarlow@redhat.com
```

Comment:
you need to add a `---` to close the front matter block

Reply:
I added the --- but still get `The YAML front matter is not present`

Reply:
Yes; there's a bug in the script. I think that's what @jmesnil meant when he said he'd fix it.

https://github.com/wildfly/wildfly-proposals/blob/main/.scripts/check_proposals#L20-L22

I expect the function needs to return after L20, otherwise it will always raise the exception at L22.

Reply:
sorry, I fix one bug and introduce another. I've fixed it now. thanks for your patience.